examples/user_guide/09-Indexing_and_Selecting_Data.ipynb

```diff
@@ -442,11 +438,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%output backend='matplotlib' size=120\n",
+    "hv.output(backend='matplotlib', size=120)\n",
+    "\n",
     "sample_style = dict(edgecolors='k', alpha=1)\n",
     "all_samples = obs_hmap.collapse().to.scatter3d().options(alpha=0.15, xticks=4)\n",
     "sampled = obs_hmap.sample((3,3))\n",
-    "subsamples = sampled.to.scatter3d().options(**sample_style)\n",
+    "subsamples = sampled.to.scatter3d().opts(**sample_style)\n",
```

Comment:
The dict should be made into keywords and should be split onto a separate line.

Reply:
They're reused at least twice.